src/server/ranch/RanchDirector.cpp

```diff
@@ -497,7 +497,16 @@ void RanchDirector::HandleEnterRanch(
         rancher.housing());
       if (housingRecords)
       {
-        protocol::BuildProtocolHousing(response.housing, *housingRecords);
+        for (const auto& housingRecord : *housingRecords)
+        {
+          housingRecord.Immutable([&response](const data::Housing& housing){
+            constexpr uint16_t IncubatorHousingId = 52;
+          // Certain types of housing have durability instead of expiration time.
+          const bool hasDurability = (housing.housingId() == IncubatorHousingId) ;
+
+          protocol::BuildProtocolHousing(response.housing.emplace_back(), housing, hasDurability);
+          });
+        }
```

Comment:
```suggestion
        for (const auto& housingRecord : *housingRecords)
        {
          housingRecord.Immutable([&response](const data::Housing& housing){
            constexpr uint16_t IncubatorHousingId = 52;
            // Certain types of housing have durability instead of expiration time.
            const bool hasDurability = (housing.housingId() == IncubatorHousingId) ;

            protocol::BuildProtocolHousing(response.housing.emplace_back(), housing, hasDurability);
          });
        }
```

fix indents